oot.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/bash
+
+while getopts ':c:' OPT; do
+        case ${OPT} in
+        c)
```

Comment:
I can't get this to work when placed after mandatory "positional" arguments. For example if I want to do `./oot/oot.sh akatsuki -c linaro_gcc`. Is this a limitation of needing `shift $(( OPTIND - 1 ))` to remove optional args or can `getopts` be used in a different way to separate flags and positional arguments?

Reply:
As far as I know it's a limitation in `getopts`.

Reply:
There are more programs where options have to forego positional arguments... If we can't come up with a solution this is fine but I will perhaps check if this is possible as I'd use that a lot.